FogCreek/Glitch-Community
# Add support for Jest in Circle CI

## Changes:
 * Fixes [Cache restore broken in CI](https://github.com/FogCreek/Glitch-Community-Work/issues/394)
 * Adds support for Jest in CircleCI - Jest will pick up tests from the `test/` directory and run them. 

## How To Test:
* Look at the CI results of this pull request - like what you see? If not, let's discuss.

* Once this is merged in: write tests using whatever chai syntax you like, put them in the `test/` directory, and run `npm run test` to see your tests run locally. When you push a branch to github, Circle CI will also run the tests for you.
